i18n/extra/es.json

```diff
@@ -93,8 +93,8 @@
             "SMW_YM"
         ],
         [
-            "SMW_MDY",
             "SMW_DMY",
+            "SMW_MDY",
```

Comment:
I made that change due to a bug I noticed every time I made semantic queries, the dates were incorrect. In ES language the dates entered  have the DMY format but the Special:Browse page or the result of some semantic queries showed the day as month (when it's < 12) and the month as day. 

I'm from Argentina. Also checked in other Spanish speaking countries and the format should be DMY as shown in this map https://en.m.wikipedia.org/wiki/Date_format_by_country